src/components/RedirectTile/RedirectTile.tsx

```diff
@@ -0,0 +1,68 @@
+import { ComponentPropsWithoutRef, ReactNode } from 'react';
+import { openInNewTab } from '../../util/utils';
+
+export interface RedirectTileProps extends ComponentPropsWithoutRef<'div'> {
+  title: string;
+  subtitle?: string;
+  description?: string;
+  logo?: ReactNode;
+  variant?: 'secondary' | 'primary' | undefined;
```

Comment:
with the `?`, you don't need undefined

```suggestion
  variant?: 'secondary' | 'primary';
```